types/twitter-stream-channels/index.d.ts

```diff
@@ -0,0 +1,134 @@
+// Type definitions for twitter-stream-channels 1.0.0
+// Project: https://github.com/topheman/twitter-stream-channels
+// Definitions by: Adrian Bardan <https://github.com/adrianbardan>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+
+/// <reference types="node" />
+/// <reference types="twit" />
+
+declare module 'twitter-stream-channels' {
+
+  import * as Twit from 'twit';
+
+  namespace TwitterStreamChannels {
+
+    export namespace StreamChannels {
+
+      export interface Channels {
+      }
+
+      export interface StreamChannelsOptions {
+        track?: Object,
+        follow?: string,
+        locations?: string,
+        enableChannelsEvents?: boolean,
+        enableRootChannelsEvent?: boolean,
+        enableKeywordsEvents?: boolean
+      }
+
+    }
+
+    export class StreamChannels extends NodeJS.EventEmitter {
+
+      /**
+       * @see https://github.com/topheman/twitter-stream-channels/blob/master/lib/StreamChannels.js#L25
+       */
+      constructor(apiClient: Twit, options: StreamChannels.StreamChannelsOptions);
+
+      /**
+       * Call this function to restart the stream after you called `.stop()` on it.
+       *
+       * Note: there is no need to call `.start()` to begin streaming. ` TwitterStreamChannels.streamChannels` calls .start() for you.
+       * @method start
+       * @returns {StreamChannels}
+       * @see https://github.com/topheman/twitter-stream-channels/blob/master/lib/StreamChannels.js#L120
+       */
+      start(): StreamChannels;
+
+      /**
+       * Closes the opened stream with Twitter
+       * @method stop
+       * @param {StreamChannelsOptions} [options]
+       * @param {StreamChannelsOptions} [options.removeAllListeners=false] If true removes all the listeners set on the stream
+       * @returns {StreamChannels}
+       * @see https://github.com/topheman/twitter-stream-channels/blob/master/lib/StreamChannels.js#L120
+       */
+      stop(options?: StreamChannels.StreamChannelsOptions): StreamChannels;
```

Comment:
only `removeAllListeners`, should not a generic "StreamChannelsOptions" interface